spec/index.bs

```diff
@@ -10200,11 +10212,9 @@ It must only be included by interfaces which also include those mixins.
                 - Let |bindGroup| be |encoder|.{{GPUBindingCommandsMixin/[[bind_groups]]}}[|index|].
                 - |bindGroup| must not be `null`.
                 - |bindGroup|.{{GPUBindGroup/[[layout]]}} must be [=group-equivalent=] with |bindGroupLayout|.
-            - For buffer bindings that weren't prevalidated with
-                {{GPUBufferBindingLayout/minBindingSize}}, the binding ranges must be large enough for
-                the [=minimum buffer binding size=].
-
-                Issue: Formalize this check.
+                - For each |entry| of |bindGroup|.{{GPUBindGroup/[[entries]]}}:
+                    - If |entry|.{{GPUBindGroupEntry/[[prevalidatedSize]]}} is `false`:
+                        - [$effective buffer binding size$](|entry|.{{GPUBindGroupEntry/resource}}) must be &ge; [=minimum buffer binding size=] of |entry|.
```

Comment:
IMO fine to be handwavy, but we should wave our hands in the direction of the shader at least. How about:

```suggestion
                        - [$effective buffer binding size$](|entry|.{{GPUBindGroupEntry/resource}}) must be &ge; [=minimum buffer binding size=]
                            of the binding variable in |pipeline|'s shader that corresponds to |entry|.
```